demo_nodes_cpp/test/parameter_events.txt

```diff
@@ -1,30 +1,42 @@
+
 Parameter event:
  new parameters:
   foo
  changed parameters:
  deleted parameters:
+
+
 Parameter event:
  new parameters:
   bar
  changed parameters:
  deleted parameters:
+
+

```

Comment:
Just curious, why two blank lines? The output I see has a log prefix, though the test passes just fine.

```
[INFO] [parameter_events]: 
Parameter event:
 new parameters:
  bar
 changed parameters:
 deleted parameters:

[INFO] [parameter_events]:
```

Reply:
the logging prefixes are not part of the processed console output so:

```
[INFO] [parameter_events]: 
```
becomes
```

```
hence the 2 blank lines.